src/commonMain/kotlin/utils/requests/FoxyRequestBuilder.kt

```diff
@@ -64,16 +102,25 @@ class FoxyRequestBuilder(
         endpoint = path
     }
 
+    /** Set the internal class FormBuilder to a private var so Foxy can use it. */
```

Comment:
This seems a little weird for documentation. Why should the developer use formData in the request?

Reply:
Done